app/.env.docker-example

```diff
@@ -7,21 +7,21 @@ DJANGO_SUPERUSER_EMAIL=admin@admin.com
 DJANGO_SUPERUSER_PASSWORD=admin
 
 # postgres settings for docker
-SQL_ENGINE=django.db.backends.postgresql
-SQL_DATABASE=people_depot_dev
-SQL_USER=people_depot
-SQL_PASSWORD=people_depot
-SQL_HOST=db
-SQL_PORT=5432
+POSTGRES_ENGINE=django.db.backends.postgresql
```

Comment:
Since the compose file only used POSTGRES USER, PASSWORD, and DB, you should just modify those 3 variable names.

I was trying to decide the best way to do this, and thinking what if we removed sqlite support to use postgres variable names only. But that's not the goal of this issue, so we should leave the code in a state where sqlite should still work.

This means that we still need to have the SQL USER, PASSWORD, and DATABASE variables since it wouldn't make sense for the sqlite section to assign to variables like POSTGRES_USER. To work around this, we can add assignments in this format for the 3 changed variables:
```
SQL_USER=POSTGRES_USER
...
```

Now we're limiting the changes to the docker section of the file and can leave the rest alone.

Please revert the changes to the local development section. I didn't give you clear enough instructions before for that, and I wasn't thinking right in terms of what this issue should be about.